template-for-repository/.gitignore

```diff
@@ -7,5 +7,9 @@ proofs/**/html
 # Emitted by CBMC Viewer
 TAGS-*
 
+# Emitted by Arpa
+arpa_cmake
```

Comment:
Since this is a directory, would `arpa_cmake/` be a more specific pattern that would match only a directory?